chapters/hlsl.adoc

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021 The Khronos Group, Inc.
+// Copyright 2021 Sascha Willems
+// SPDX-License-Identifier: CC-BY-4.0
+
+ifndef::chapters[:chapters:]
+
+[[hlsl-in-vulkan]]
+= HLSL in Vulkan
+
+Vulkan does not directly consume shaders in a human-readable text format, but instead uses link:./what_is_spirv.adoc[SPIR-V] as an intermediate representation. This opens the option to use shader languages other than e.g. GLSL, as long as they can target the Vulkan SPIR-V environment.
+
+One such language is the High Level Shading Language (HLSL) by Microsoft, used by DirectX. Thanks to link:https://www.khronos.org/blog/hlsl-first-class-vulkan-shading-language[recent additions to Vulkan 1.2] it is now considered a first class shading language for Vulkan that can be used just as easily as GLSL.
+
+With link:https://github.com/microsoft/DirectXShaderCompiler/blob/master/docs/SPIR-V.rst#unsupported-hlsl-features[a few exceptions], all Vulkan features and shader stages available with GLSL can be used with HLSL too, including recent Vulkan additions like hardware accelerated ray tracing. On the other hand, HLSL to SPIR-V supports Vulkan exclusive features that are not (yet) available in DirectX.
+
+image::images/what_is_spirv_dxc.png[what_is_spriv_dxc.png]
+
+[[applications-pov]]
+= From the application's point-of-view
```

Comment:
nit: each chapter should only have one `H1` header so just add an extra `=` to the rest of the headers in this chapter